### ISSUE TITLE
no implicit conversion of nil into String in `title`

### ISSUE THREAD

**Issue by parkr** [jekyll/jekyll-seo-tag#207]
This line: https://github.com/jekyll/jekyll-seo-tag/blob/96b1fa22c5fc852a7f7116f0792edd5d1de255ef/lib/jekyll-seo-tag/drop.rb#L45 does:

```ruby
site_title + TITLE_SEPARATOR + format_string(site["description"])
```

One of those three is `nil`. Can we cast them all to a string or do a `[...].compact.join("")`?

**Comment by pathawks** [comment]
See also #206

**Comment by pathawks** [comment]
Fixed by #206
